src/hastings_rpc.erl

```diff
@@ -111,8 +111,8 @@ get_update_seq(DbName) ->
 get_or_create_db(DbName, Options) ->
     case couch_db:open_int(DbName, Options) of
         {not_found, no_db_file} ->
-            couch_log:warning("~s creating ~s", [?MODULE, DbName]),
-            couch_server:create(DbName, Options);
+            couch_log:warning("~p creating ~s", [?MODULE, DbName]),
+            mem3_util:get_or_create_db(DbName, Options);
```

Comment:
We'd use the _int version here I think?

Reply:
No `_int` here because this was using `couch_server:create` which performs the `couch_server:validate_open_or_create` logic that `couch_server:open` also does.

Reply:
Aha, makes sense!